compose-Satosa-Saml2Spid/nginx/nginx.conf_uwsgi_pass

```diff
@@ -0,0 +1,105 @@
+user nginx;
```

Comment:
this is quite duplicated considering [nginx.conf_proxy_http](https://github.com/italia/Satosa-Saml2Spid/pull/55/files#diff-14c1e64cc382127f1fd94d1add1c93382a1c7a7a916d476a3a4a7ec41cb0f995)

please clean up or explain the rationale for this

Reply:
Non vengono utilizzate entrambe le configurazioni, ma solo una. 
La mia scelta è stata quella di usare nginx.conf_uwsgi_pass come default. Ma di dare anche a chi preferisce il proxy http una configurazione già fatta da poter utilizzare. Credo di averlo spiegato in un qualche punto della documentazione (forse è accennato solo in un commento nel run.sh), devo ricontrollare ed in caso aggiungerlo.
https://github.com/IDEM-GARR-AAI/Satosa-Saml2Spid/blob/b70b650bfeb3e51ff6b794e22bbbf172339c3715/example/run.sh#L179
Il fatto è che secondo me ci sono vantaggi e svantaggi in entrambe le soluzioni, ma quella più adatta a "produzione" mi sembra quella che utilizza uwsgi_pass.
L'immagine creata con il Dockerfile ha come default, nel run.sh , l'avvio di uwsgi con --socket (che comunque dovrebbe essere il default) e questo impone di utilizzare in nginx  uwsgi_pass. In questo modo uwsgi dovrebbe essere più veloce, perché non deve fare il lavoro extra di tradurre in http, tanto poi a parlare in http e https ci pensa nginx.
Chi invece vuole usare la configurazione proxy_http deve cambiare la configurazione in example/run.sh , oppure sostituire nel container running il file run.sh , con la configurazione modificata, andando ad aggiungerlo nel docker-compose.yml.
Possiamo pure pensare di scegliere solo una delle due possibilità e fornire quindi una sola configurazione.

Reply:
ok, la tua scelta mi torna

magari sarà più difficile da mantenere ma non importa dato che queste configurazioni hanno lo scopo di esemplificare il deployment